src/libs/retry/infinite.ts

```diff
@@ -0,0 +1,28 @@
+import { Runtime } from '@lib/runtime/runtime';
+
+import { BackOff } from './backoff/backoff';
+import { Retry } from './retry';
+
+export class Infinite implements Retry {
```

Comment:
```suggestion
export class InfiniteRetry implements Retry {
```

TypeScript won't automatically prefix package name in front of class name. To make our classes more clear, let's name it to InfiniteRetry